modules/apps/object/object-test/src/testIntegration/java/com/liferay/object/service/test/ObjectEntryLocalServiceTest.java

```diff
@@ -1455,6 +1455,13 @@ public void testAddObjectEntryWithObjectValidationRule() throws Exception {
 
 		DateTimeFormatter dateTimeFormatter = DateTimeFormatter.ofPattern(
 			"yyyy-MM-dd HH:mm");
+		LocalDateTime tomorrowLocalDateTime = LocalDateTime.now(
+		).plusDays(
+			1
+		);
+
+		String formattedLocalDateTime = dateTimeFormatter.format(
+			tomorrowLocalDateTime);
```

Comment:
Hey @diogomelo09,

Avoid chaining:

```suggestion
		LocalDateTime localDateTime = LocalDateTime.now();

		String formattedLocalDateTime = dateTimeFormatter.format(
			localDateTime.plusDays(1));
```

Rename to `localDateTimeString` because the pattern is define the type of the variable as a suffix:

```suggestion
		LocalDateTime localDateTime = LocalDateTime.now();

		String localDateTimeString = dateTimeFormatter.format(
			localDateTime.plusDays(1));
```

Thanks!

Reply:
Done!! :heavy_check_mark: